docker/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM quay.io/sameersbn/ubuntu:latest
+MAINTAINER Alexander Trost <galexrt@googlemail.com>
+
+ENV ZULIP_VERSION="1.3.7" ZULIP_CHECKSUM="88bfa668eb14e07b0b806977db2ae2cd4d7e7ef8" DATA_DIR="/data"
```

Comment:
By hardcoding this hash, we have an unfortunate chicken-and-egg problem where a tarball that includes `docker/` cannot possibly be the same as the release tarball. 

We'd need to exclude `docker/*` from the release tarball and distribute it separately, or use the existing unpacked codebase (if that's something Docker supports?)